console/console-init/ui/src/modules/address-detail/AddressDetailPage.tsx

```diff
@@ -228,6 +230,10 @@ export default function AddressDetailPage() {
           name={address.name}
           plan={address.planLabel}
           topic={address.topic}
+          Deadletters={address.Deadletter}
```

Comment:
please revisit this name. All props will be camel cased. like `deadLetter` and `expiryQueue`